integrationtests/gomodvendor/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/lucas-clemente/quic-go/integrationtets/gomodvendor
```

Comment:
best to not make this module public/importable, to prevent confusion. you could name it `module test`, as that's a module path prefix reserved specifically for this kind of thing.

Reply:
Good point. Didn't know that this was an option :)